code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -51,11 +51,13 @@
 
 /obj/structure/beebox/Destroy()
 	processing_objects.Remove(src)
+	for(var/mob/living/simple_animal/hostile/poison/bees/B in bees)
+		B.beehome = null
 	bees.Cut()
 	bees = null
-	honeycombs.Cut()
-	honeycombs = null
-	queen_bee = null
+	QDEL_LIST(honeycombs)
+	QDEL_LIST(honey_frames)
+	QDEL_NULL(queen_bee)
```

Comment:
@RemieRichards Why you no clear all references.

On that note, this makes beeboxes GC more consistently, but there's still some issues, once in a while; any ideas on what else needs its reference cleared out?